src/assets/index.ts

```diff
@@ -1,19 +1,31 @@
 import { AssetInfo } from "../interface";
 
+const environment =
+  process.env.REACT_APP_STAGE || process.env.ENVIRONMENT || "";
```

Comment:
why do we have two env vars for specify environment? Can we use only one?

Reply:
both the UI and the rest server use the SDK. in the UI, the environment is called REACT_APP_STAGE, and it's just ENVIRONMENT in the rest server

Reply:
I see, any reason why we can't use the same var for both?

Reply:
react apps only recognize environment variables with the "REACT_APP_" prefix. so we could also change both the backend and frontend environment variables to be something like "REACT_APP_ENVIRONMENT", but that would be a weird convention for the backend.